about.html

```diff
@@ -0,0 +1,203 @@
+      integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./css/about.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <!-- HEADER SECTION -->
+    <header class="header">
+      <div class="header-links">
+        <div class="header-left"></div>
+        <div class="header-right">
+          <ul class="header-link">
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fab fa-facebook"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fab fa-twitter"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fa-brands fa-linkedin-in"></i>
+              </a>
+            </li>
+            <li>
+              <a href="https://www.twitter.com">
+                <i class="fa-brands fa-instagram"></i>
+              </a>
+            </li>
+          </ul>
+          <div class="header-items">
+            <a href="#">English</a>
+            <a href="#">Logout</a>
+          </div>
+        </div>
+      </div>
+      <!-- NAV SECTION -->
+      <nav>
+        <div class="nav-center">
+          <div class="nav-header">
+            <div class="logo">
+              <a href="index.html">
+                <img
+                  src="./assets/Images/conference-logo.png"
+                  alt="conference"
+                />
+              </a>
+            </div>
+            <div class="nav-toggle">
+              <i class="fa-solid fa-bars"></i>
+              <i class="fa-solid fa-times"></i>
+            </div>
+          </div>
+          <ul class="links">
+            <li class="home"><a href="index.html">Home</a></li>
+            <li class="about"><a href="#">About</a></li>
+            <li class="about"><a href="#join">Join</a></li>
+          </ul>
+        </div>
+      </nav>
+
+      <!-- HERO SECTION -->
+      <section class="headline-container">
+        <div class="headline">
+          <div class="hero-header-title">
+            <h3>"Hello! Sharing World"</h3>
+          </div>
+          <div class="hero-title">
+            <h1>International Conference on Sustainable Water Management</h1>
+          </div>
+          <div class="hero-desc">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaerat
+              culpa illum et neque veritatis, non itaque quasi deleniti nostrum
+              nesciunt. Lorem ipsum dolor sit amet consectetur, adipisicing
+              elit. Velit vel, iusto nostrum explicabo veritatis sint nisi vitae
+              placeat officiis doloremque, soluta officia ut perspiciatis
+              quibusdam quos nam exercitationem optio adipisci.
+            </p>
+          </div>
```

Comment:
- [ ] This text box is not centered on the page on screens with a width between 480px and 1200px due to a margin. Kindly remove that margin so this `div` can be centered and looks more professional like the template.
 - Your design: 
 - 
<img width="889" alt="image" src="https://user-images.githubusercontent.com/61919617/203633371-d939d861-9574-4c1c-b4de-7cf4e3102dc5.png">